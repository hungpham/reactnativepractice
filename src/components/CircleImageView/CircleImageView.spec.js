import 'react-native';
import React from 'react';
// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

import CircleImageView from './CircleImageView';

it('CircleImageView renders correctly', () => {
  const tree = renderer.create(
    <CircleImageView />
  ).toJSON();
	expect(tree).toMatchSnapshot()
});
