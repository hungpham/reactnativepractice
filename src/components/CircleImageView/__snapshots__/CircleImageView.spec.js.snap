// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CircleImageView renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "backgroundColor": "white",
        "borderRadius": 45,
        "height": 90,
        "justifyContent": "center",
        "width": 90,
      },
      Object {
        "borderRadius": NaN,
        "height": undefined,
        "width": undefined,
      },
    ]
  }
>
  <Image
    source={undefined}
    style={
      Object {
        "height": 40,
        "width": 40,
      }
    }
  />
</View>
`;
